pkg/cmd/pr/view/view.go

```diff
@@ -201,22 +211,25 @@ func printHumanPrPreview(opts *ViewOptions, pr *api.PullRequest) error {
 	}
 
 	// Body
-	fmt.Fprintln(out)
+	var md string
+	var err error
 	if pr.Body == "" {
-		pr.Body = "_No description provided_"
-	}
-	style := markdown.GetStyle(opts.IO.TerminalTheme())
-	md, err := markdown.Render(pr.Body, style, "")
-	if err != nil {
-		return err
+		md = fmt.Sprintf("\n  %s\n\n", cs.Gray("No description provided"))
+	} else {
+		style := markdown.GetStyle(opts.IO.TerminalTheme())
+		md, err = markdown.Render(pr.Body, style, "")
+		if err != nil {
+			return err
+		}
 	}
+	fmt.Fprintln(out)
 	fmt.Fprint(out, md)
 	fmt.Fprintln(out)
```

Comment:
These lines could get condensed into a single Fprintf statement. Or, individual Fprintf statements could get inlined into `if pr.Body == "" ... else` blocks so it's easier to follow how whitespace is added around output.